ci/windows/test_jl10_cpu.ps1

```diff
@@ -0,0 +1,54 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+7z x -y windows_package.7z
+
+# set default output encoding to utf8
+$PSDefaultParameterValues['Out-File:Encoding'] = 'utf8'
+
+$env:MXNET_HOME = [System.IO.Path]::GetFullPath('.\windows_package')
+$env:JULIA_URL = "https://julialang-s3.julialang.org/bin/winnt/x64/1.0/julia-1.0.3-win64.exe"
+$env:JULIA_DEPOT_PATH = [System.IO.Path]::GetFullPath('.\julia-depot')
+
+$JULIA_DIR = [System.IO.Path]::GetFullPath('.\julia10')
+$JULIA = "$JULIA_DIR\bin\julia"
+
+# Download most recent Julia Windows binary
+[System.Net.ServicePointManager]::SecurityProtocol = [System.Net.SecurityProtocolType]::Tls12
+(New-Object System.Net.WebClient).DownloadFile($env:JULIA_URL, "julia-binary.exe")
+if (! $?) { Throw ("Error on downloading Julia Windows binary") }
+
+# Run installer silently, output to C:\julia10\julia
+Start-Process -Wait "julia-binary.exe" -ArgumentList "/S /D=$JULIA_DIR"
+if (! $?) { Throw ("Error on installing Julia") }
+
+& $JULIA -e "using InteractiveUtils; versioninfo()"
+
+# workaround a bug of Julia 0.7
+# use PackageSpec(name = "MXNet", path = "...") if we drop the 0.7 support
+Copy-Item -Path .\julia -Recurse -Destination "$env:JULIA_DEPOT_PATH\dev\MXNet"
```

Comment:
It seems like it's this line @iblis17

Reply:
![screenshot](https://user-images.githubusercontent.com/761623/51526253-e8a5ec00-1e6c-11e9-8e57-507e1e45da50.PNG)

I found the julia function `Pkg.develop` cause that error message, but it's intended.

I'm going to find another way to install MXNet package locally.

Reply:
Seems powershell will capture `stderr` and raise `NativeCommandError`, since Julia generate the logger output to `stderr`. (FYI: https://superuser.com/questions/1253297)

```
PS C:\Users\iblis\tmp\incubator-mxnet\ci\windows> echo '@info "test"' | & $JULIA
julia.exe : [ Info: test
位於 線路:1 字元:23
+ echo '@info "test"' | & $JULIA
+                       ~~~~~~~~
    + CategoryInfo          : NotSpecified: ([ Info: test:String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
```

`@info "test"` is a valid Julia logger syntax.
```julia
julia> @info "test"
[ Info: test 

```

The solution is redirecting them to `stdout`.